client/termsetting/termsetting.ts

```diff
@@ -847,18 +847,46 @@ type DefaultQByTsHandler = {
 	snplst?: SnpsQ
 }
 
-export async function fillTermWrapper(tw: TermWrapper, vocabApi: VocabApi, defaultQByTsHandler?: DefaultQByTsHandler) {
+export async function fillTwLst(twlst: TwLst, vocabApi: VocabApi, defaultQByTsHandler?: DefaultQByTsHandler): void {
+	const dictTerms = await getDictTerms(twlst, vocabApi)
+	const promises: Promise[] = []
```

Comment:
Should be `Promise<TermWrapper>`

Reply:
@airenzp could you help fix these type errors and commit? my code editor is not as helpful in finding these

Reply:
Done